Add Libvirt container and compile the installer

gbenhaim opened the pull request.

gbenhaim committed:
Adding container with installer dependencies

The added container can be used to run the installer.
Note: This container doesn't include the required deps for compiling the
installer.

Signed-off-by: gbenhaim <galbh2@gmail.com>

gbenhaim committed:
Build the installer

Added a build container for building the installer.

Signed-off-by: gbenhaim <galbh2@gmail.com>

Galbh2 commented:
@cynepco3hahue any idea how do I fix the import path?
The vendor is being ignored.

https://jenkins.ovirt.org/job/no-name-yet_installer_standard-check-pr/6//artifact/check-patch.el7.x86_64/mock_logs/script/stdout_stderr.log

cynepco3hahue commented:
the absolute path of the installer must be
`/root/go/src/github.com/openshift/installer` in case if you run under the root

gbenhaim committed:
Run the installer inside a container

Signed-off-by: gbenhaim <galbh2@gmail.com>

gbenhaim committed:
Move NetwormManager and virsh config to runtime

Configuring NetwrokManager during build time can damage DNS
configuration since we don't now what will be the DNS servers
when the container will be started.

Since only virsh configuration was left in setup.sh, move it also
to runtime so we can remove setup.sh.

Signed-off-by: gbenhaim <galbh2@gmail.com>

gbenhaim committed:
Allow ssh access to the cluster VMs and collect booktube log

Signed-off-by: gbenhaim <galbh2@gmail.com>

gbenhaim commented:
ci test please

gbraad reviewed (commented):
s/booktube/bootkube/

gbraad reviewed (commented):
s/NetwormManager/NetworkManager/
